model/project_ref.go

```diff
@@ -61,6 +63,10 @@ type RepositoryErrorDetails struct {
 	MergeBaseRevision string `bson:"merge_base_revision" json:"merge_base_revision"`
 }
 
+type FilesIgnoredFromCache struct {
+	FilePatterns string `bson:"file_patterns" json:"file_patterns"`
```

Comment:
Should this just be a list? Why is this nested, and not just a value above?

Reply:
I thought you had mentioned something about nesting this in a struct. I must have misunderstood. I've switched to just using a list.